catalogs/catalog-jdbc-mysql/src/main/java/com/datastrato/gravitino/catalog/mysql/operation/MysqlDatabaseOperations.java

```diff
@@ -13,13 +13,28 @@
 import java.sql.ResultSet;
 import java.sql.SQLException;
 import java.sql.Statement;
+import java.util.Collections;
 import java.util.HashMap;
+import java.util.HashSet;
 import java.util.Map;
+import java.util.Set;
 import org.apache.commons.collections4.MapUtils;
 import org.apache.commons.lang3.StringUtils;
 
 /** Database operations for MySQL. */
 public class MysqlDatabaseOperations extends JdbcDatabaseOperations {
+
+  public static final Set<String> SYS_MYSQL_DATABASE_NAMES =
+      Collections.unmodifiableSet(
+          new HashSet<String>() {
```

Comment:
I would suggest introduce immutable lib and use `ImmutableSet.of(...)` directly, It's quite tedious here.